packages/team-management/src/ducks/practitioners.ts

```diff
@@ -0,0 +1,54 @@
+/** Organizations redux module */
+import {
```

Comment:
What is this ducks module for? I don't see any actions/selectors that defined here being used anywhere in this PR.